diff --git a/external/cmake/dependencyinfo.cmake b/external/cmake/dependencyinfo.cmake
index b456d1e1..ed69ef47 100644
--- a/external/cmake/dependencyinfo.cmake
+++ b/external/cmake/dependencyinfo.cmake
@@ -6,14 +6,7 @@ endif()
 
 get_filename_component(DEPENDENCIES_INSTALL_DIR ${DEPENDENCIES_INSTALL_DIR}/${CMAKE_SYSTEM_NAME} ABSOLUTE)
 
-if (WIN32)
-    set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>$<$<BOOL:BUILD_SHARED_LIBS>:DLL>")
-    if (BUILD_SHARED_LIBS)
-        get_filename_component(DEPENDENCIES_INSTALL_DIR ${DEPENDENCIES_INSTALL_DIR}/MD ABSOLUTE)
-    else()
-        get_filename_component(DEPENDENCIES_INSTALL_DIR ${DEPENDENCIES_INSTALL_DIR}/MT ABSOLUTE)
-    endif()
-elseif (NOT APPLE)
+if (NOT APPLE)
     if (EXISTS /etc/os-release)
         file(STRINGS /etc/os-release ID_FIELD
             REGEX "^ID=.*$"
